specs/lightclient/beacon-chain.md

```diff
@@ -235,13 +228,20 @@ def eth2_fast_aggregate_verify(pubkeys: Sequence[BLSPubkey], message: Bytes32, s
     return bls.FastAggregateVerify(pubkeys, message, signature)
 ```
 
-#### `is_activation_exit_period_boundary`
+#### `is_activation_exit_epoch`
 
 ```python
-def is_activation_exit_period_boundary(state: BeaconState) -> bool:
+def is_activation_exit_epoch(state: BeaconState) -> bool:
     return get_current_epoch(state) % EPOCHS_PER_ACTIVATION_EXIT_PERIOD == 0
 ```
 
+#### `is_leak_active`
+
+```python
+def is_leak_active(state: BeaconState) -> bool:
+    return get_finality_delay(state) > EPOCHS_TO_LEAK_PENALTIES
+```
```

Comment:
It's a duplicate of the phase 0 version.

Reply:
leaving in until we decide on cross-cutting changes on phases